### PR TITLE
chore: Bump up model-server to v0.7.11-2

### PR DESCRIPTION
**Pull request by vprashar2929**
This commit bumps up model-server to `v0.7.11-2`

Addresses: #420

### CODE REVIEW

File: pkg/components/estimator/estimator.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 
 const (
 	// NOTE: update tests/images.yaml when changing this image
-	StableImage          = "quay.io/sustainable_computing_io/kepler_model_server:v0.7.11"
+	StableImage          = "quay.io/sustainable_computing_io/kepler_model_server:v0.7.11-2"
```
Comment (sthaha) [inline pkg/components/estimator/estimator.go:31]: @vprashar2929 we should also change the entrypoint and command
Please check the model-server compose files.

Comment (vprashar2929) [reply pkg/components/estimator/estimator.go:31]: Yeah forgot about that 😅

File: pkg/components/estimator/estimator.go [hunk 2]
```diff
@@ -76,8 +76,8 @@ func Container(image string) corev1.Container {
 		ImagePullPolicy: corev1.PullIfNotPresent,
 		Name:            "estimator",
 		VolumeMounts:    mounts,
-		Command:         []string{"python3.10"},
-		Args:            []string{"-u", "src/estimate/estimator.py"},
+		Command:         []string{"estimator"},
+		Args:            []string{"-l", "info"},
```
Comment (sthaha) [inline pkg/components/estimator/estimator.go:80]: `-l info` is optional btw ...

**Review by sthaha — CHANGES_REQUESTED** [review]
We should also update kepler: command to wait for both model-server and estimator

**Review by sthaha — APPROVED** [review]

**Pull request merged**